ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productteaser/v1/productteaser/_cq_editConfig.xml

```diff
@@ -7,6 +7,11 @@
             jcr:primaryType="cq:DropTargetConfig"
             accept="[.*]"
             groups="[product]"
-            propertyName="./selection"/>
+            propertyName="./multiDropTarget->@selection">
```

Comment:
Why needs the product teaser the "multiDropTarget"?

Reply:
Because we implement that DnD "save" feature in the `MultiFieldDropTargetPostProcessor` so we don't have to introduce another post-processor. I added the `multiple` parameter to control the ADD or REPLACE behaviour, so the teaser uses `false` so DnD'ing another product replaces the selection.

Reply:
+1 for keeping everything in on post-processor. But this looks cumbersome somehow because it is not "AEM standard".

Reply:
I wonder if we can find a way to get both property flavors: `propertyName="./selection"` and  `propertyName="./multiDropTarget->@product"` working?

The first format is the AEM "standard" and ideal for components which do only accept d&d of one product this should be used.

Reply:
Well if we want to allow the "standard" DnD **AND** the choice of `selectionId` (sku, path, slug), we must have a way to "detect" that special DnD feature. A possible alternative to what is being done here could be:
```
propertyName = "./selection"
<parameters
  selectionId="multiDropTargetValue->sku@selection">
```
We need 2 things:
- we must "indicate" that this POST must be processed by the `MultiFieldDropTargetPostProcessor` (so we need a special `String` somewhere to "activate" the post processor)
- we have to indicate which property will hold the value. This is required because the DnD POST sets all properties in the same HTTP form, so we have to make sure we know which property must be modified before saving.

So I think it's possible, but would add more complexity to the post-processor, and anyway requires a special configuration in `cq:editConfig` ... so I don't really see a big benefit. If someone only needs the standard existing DnD, they can just continue to use this.

Reply:
Ok, I think I can agree, let's keep the current approach then.